src/utils/common_utils.hpp

```diff
@@ -81,9 +81,13 @@ bool make_path(const std::string& path);
 /// Check if directory with given path exist
 bool is_dir_exist(const std::string& path);
 
+/// Enum to wrap bool variable to select if random string
+/// should have numbers or not
+enum UseNumbersInString : bool { WithNumbers = true, WithoutNumbers = false };
+
 /// Generate random std::string of length len based on a
 /// uniform distribution
-std::string generate_random_string(int len);
+std::string generate_random_string(int len, UseNumbersInString use_numbers);
```

Comment:
@iomaganaris : we were discussing to keep here default argument `WithoutNumbers` so that all other API usage remains same i.e. no modifications?

Reply:
`generate_random_string` is only used in one place, inside the `SingletonRandomString`, and needs to get this variable as a parameter to select if it will add numbers or not, so this is why I kept my change in this signature